common/djangoapps/student/management/commands/manage_user.py

```diff
@@ -15,6 +15,23 @@
 from student.models import UserProfile
 
 
+def is_valid_django_hash(encoded):
+    """
+    Starting with django 2.1, the function is_password_usable no longer checks whether encode
+    is a valid password created by a django hasher(hasher in  PASSWORD_HASHERS setting)
+
+    Adding this function to create constant behavior as we upgrade django versions
+    """
+    if encoded is None:
+        return False
+
```

Comment:
FYI: This doesn't hurt, but I don't think you need this because we already check `initial_password_hash` is not None.  Not a big deal either way.